tests/api/Jalasoft.TeamUp.Resumes.API.Tests/PostResumesTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Jalasoft.TeamUp.Resumes.API.Tests
+{
+    using System;
+    using Jalasoft.TeamUp.Resumes.API.Controllers;
+    using Jalasoft.TeamUp.Resumes.Core;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.DAL.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using Xunit;
+
+    public class PostResumesTests
+    {
+        private readonly ResumesService resumeService;
+        private readonly Mock<IResumesRepository> mockRepository;
+
+        public PostResumesTests()
+        {
+            this.mockRepository = new Mock<IResumesRepository>();
+            this.resumeService = new ResumesService(this.mockRepository.Object);
+        }
+
+        [Fact]
```

Comment:
please add an Assert to verify the response code is the correct (200) one and also please create negative use test where the object could not be created and the response code is an error (adding validations in the service layer)

Reply:
Also add unit tets for the service layer

Reply:
Done